app/views/passengers/_contact_information.haml

```diff
@@ -0,0 +1,7 @@
+= verification.name
```

Comment:
The file name of this partial might lead me to believe that it contained the contact information of the passenger.